src/vs/workbench/contrib/chat/browser/chat.contribution.ts

```diff
@@ -413,12 +413,12 @@ class ChatAgentSettingContribution extends Disposable implements IWorkbenchContr
 
 		const expDisabledKey = ChatContextKeys.Editing.agentModeDisallowed.bindTo(contextKeyService);
 		experimentService.getTreatment<boolean>('chatAgentEnabled').then(enabled => {
-			if (enabled) {
+			if (enabled || typeof enabled !== 'boolean') {
+				// If enabled, or experiments not available, fall back to registering the setting
 				this.registerEnablementSetting();
 				expDisabledKey.set(false);
```

Comment:
If experiments are disabled and we report them as enabled here, might that cause another issue?
Approving in the meantime because I agree with the idea behind the candidate discussed in the meeting.